lib/iiif/v3/abstract_resource.rb

```diff
@@ -51,7 +51,7 @@ def numeric_only_keys
       end
 
       def uri_only_keys
-        %w{ }
+        %w{ rights }
```

Comment:
This can be a string or a URI. It looks like the extension mechanism doesn't requite a URI from what I can tell